### PR TITLE
trace #19385

### PR DESCRIPTION
**Pull request by hopedonn**
Close https://github.com/unifyai/ivy/issues/19385

### CODE REVIEW

**Comment by hopedonn** [comment]
Close #19385

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

**Comment by hopedonn** [comment]
@MahmoudAshraf97

File: ivy/functional/frontends/paddle/tensor/math.py [hunk 1]
```diff
@@ -346,3 +346,13 @@ def frac(x, name=None):
...
+
+@with_supported_dtypes(
+    {"2.5.0 and below": ("float32", "float64", "int32", "int64")},
+    "paddle",
+)
+@to_ivy_arrays_and_back
+def trace(x, offset=0, axis1=0, axis2=0, name=None):
+    return ivy.trace(x)
+
+
```
Comment (xoiga123) [inline ivy/functional/frontends/paddle/tensor/math.py:358]: Please follow the formatting convention of 2 blank lines between function definitions and a newline at end-of-file.

**Review by xoiga123 — CHANGES_REQUESTED** [review]
Thank you for contributing to Ivy :hugs: Need some changes here and there, but looking good overall :+1:

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.

**Pull request closed without merging**